src/DIRAC/FrameworkSystem/DB/ComponentMonitoringDB.py

```diff
@@ -384,7 +384,8 @@ def getComponentsStatus(self, conditionDict={}):
         # Walk the URLs
         serviceURLs = getSystemURLs(system, setup)  # verify URLs in getSystemURLs method
         for service in serviceURLs:
-          url = urlparse.urlparse(serviceURLs[service])
+          # serviceURLs is a dict that contain a list of URLs for service
+          url = urlparse.urlparse(serviceURLs[service][0])
```

Comment:
so here you arbitrarily take the first one ?

Reply:
Yes, from randomised list